content/get-started/ledger.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: Ledger
+
+---
+
+A Ledger is a hardware wallet used to securely store your crypto assets offline. The private key used to sign transactions never leaves the Ledger device, so even if your machine or wallet is compromised, your private key is never leaked. 
+
+Using Ledger for Iron Fish will enable you to manage IRON as well as any other Iron Fish asset on the Ledger device. There are two flows that will be described in this tutorial: the single-signer flow, which is the most common type of wallet, or the multi-sig wallet flow allowing a wallet to be controlled by a number of participants. 
+
+If you have any questions on how to use your Ledger for Iron Fish assets, please reach out on our [Discord](https://discord.ironfish.network/) in the questions channel.  
+
+## Video Walkthrough Tutorial
+
+The video walkthrough tutorial for both single-signer flow using the Ironfish Ledger app as well as the multi-sig wallet flow using the Ironfish DKG app can be found here: 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ELBw9I1AVrk?si=TM3qJzGzSRlp5uQF" title="YouTube video player" frameBorder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerPolicy="strict-origin-when-cross-origin" allowFullScreen></iframe>
+
+## Single Signer Ledger Flow
+
+### Requirements
+
+You’ll need the latest version of the [Node App](https://ironfish.network/use/node-app) to see the balance, transaction history, and facilitate creating transactions for your Ledger wallet.  Please make sure to always have your Iron Fish [Node App](https://ironfish.network/use/node-app) up to date. Installing a new version will preserve all your chain and wallet history. 
+
+For your Ledger device, make sure to: 
+
+- Have [Ledger Live installed](https://support.ledger.com/article/4404389503889-zd)
```

Comment:
```suggestion
- Install [Ledger Live](https://support.ledger.com/article/4404389503889-zd)
```

Reply:
I would keep this simple since it's an instruction to the user